src/components/Dropdown.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import './dropdown.css';
+
+class Dropdown extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {display: 'Home'};
+        this.handleOption = this.handleOption.bind(this);
```

Comment:
esti sigur ca mai ai nevoie aici de .bind(this)?

Daca la un class component declari method-ul like this: `methodName = () => {}` atunci nu mai o sa ai nevoie sa bind this la method. Daca ai declara method-ul `methodName() {}`, atunci da, trebuie sa bind this la method. ✌️